src/main/java/uk/sky/cqlmigrate/CqlLoader.java

```diff
@@ -26,9 +31,9 @@ static void load(Session session, Path cqlPath) {
             cqlReader.lines().forEach(statementBuilder::append);
 
             String cqlStatements = statementBuilder.toString();
-            checkState(cqlStatements.endsWith(";"), "had a non-terminated cql line: %s", cqlStatements);
+            checkState(cqlStatements.endsWith(CQL_STATEMENT_TERMINATOR), "had a non-terminated cql line: %s", cqlStatements);
```

Comment:
This looks to throw an exception just be caught below as we're modifying this file we should clean this up